pkg/metrics/metrics.go

```diff
@@ -61,12 +62,22 @@ var CertificateExpiryTimeSeconds = prometheus.NewGaugeVec(
 	[]string{"name", "namespace"},
 )
 
+var CertificateIssuedLatency = prometheus.NewHistogram(
+	prometheus.HistogramOpts{
+		Namespace: namespace,
+		Name:      "certificate_issued_latency",
```

Comment:
IIRC, Prometheus metrics should have the unit as part of the name? i.e. `certificate_issue_latency_seconds`?